content/_includes/CLI/AccountGroupDeleteExample.md

```diff
@@ -0,0 +1,17 @@
+---
+---
+
+```
+account group delete id=<GID>
+```
+
+Where `<GID>` is the group identification number.
```

Comment:
Same comment as for the command string. If the user is expected to enter a number value then enter an example GID as a number, then explain that number is the GID.